### PR TITLE
Add --all flag to delete all locally available models

### PR DESCRIPTION
**Pull request by JoyProg**
**Description**

This feature enhances Ersilia's delete command by adding the functionality to delete multiple models.

**Changes Made**

 - Implemented a delete all models feature.
- Refactored the code to eliminate redundancy, ensuring both single and multiple model deletions now call a base method by passing in the model ID.

Related to #1265

### CODE REVIEW

**Comment by P-Mefut** [comment]
Hi @JoyProg could you please share your logs output, I've tried running test and I'm getting a no such option for the --all tag

[output.log](https://github.com/user-attachments/files/17330136/output.log)

**Comment by JoyProg** [comment]
Hi @P-Mefut. I do not get the error. I've attached the log file.  Can you verify if the file you have locally reflects the changes? 
[output.log](https://github.com/user-attachments/files/17330484/output.log)

**Comment by JoyProg** [comment]
Hi @P-Mefut, are you still getting the same output?

**Comment by P-Mefut** [comment]
Hi @DhanshreeA I've checked again and everything is in good standing, the code works just fine👍

File: ersilia/cli/commands/delete.py [hunk 1]
```diff
@@ -37,3 +29,41 @@ def delete(model):
...
+            if _delete_model_by_id(model_id):
+                deleted_count += 1
+        echo(":thumbs_up: Completed the deletion of all locally available models!", fg="green")
+        
+
+    # Example usage:
+    # 1. Delete a specific model: ersilia delete {MODEL}
+    # 2. Delete all models: ersilia delete --all
+    @ersilia_cli.command(
+        short_help="Delete model from local computer",
```
Comment (DhanshreeA) [inline ersilia/cli/commands/delete.py:56]: Maybe we can rephrase this better, something like, "Delete model(s) available locally"

File: ersilia/cli/commands/delete.py [hunk 2]
```diff
@@ -37,3 +29,41 @@ def delete(model):
...
+        help="Delete model from local computer. The BentoML bundle is deleted, as well as the files stored in "
```
Comment (DhanshreeA) [inline ersilia/cli/commands/delete.py:57]: "Delete model(s) available locally. The BentoML bundle is deleted, as well as the files stored in the EOS directory."

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

**Comment by dzumii** [comment]
Hello @JoyProg have you implemented the requested changes here?

**Review by DhanshreeA — APPROVED** [review]

**Pull request merged**